pipeline/merger/src/main/scala/weco/pipeline/merger/services/Merger.scala

```diff
@@ -176,14 +177,26 @@ object Merger {
   }
 }
 
-object PlatformMerger extends Merger {
-  import weco.catalogue.internal_model.image.ParentWork._
+object DefaultPlatformMerger extends BasePlatformMerger {
+  override val targetPrecedence: BaseTargetPrecedence = DefaultTargetPrecedence
+  override val passThroughTransform: PassThroughTransform =
+    DefaultPassThroughTransform$
+}
+
+object TeiPlatformMerger extends BasePlatformMerger {
+  override val targetPrecedence: BaseTargetPrecedence = TeiTargetPrecedence
+  override val passThroughTransform: PassThroughTransform =
+    TeiPassThroughTransform$
+}
```

Comment:
Again, the approach of putting specialised implementations ahead of the base trait feels weird to me when reading this file.

Reply:
In this instance, I imagine it's just about editing the file for this work, it's easy to have the TEI (what's in focus) at the top of the file. Agree generally I'd go with the trait => trait implementations